crates/types/src/lib.rs

```diff
@@ -0,0 +1 @@
+include!(concat!(env!("OUT_DIR"), "/blocks.rs"));
```

Comment:
```suggestion
pub mod blocks {
    include!(concat!(env!("OUT_DIR"), "/blocks.rs"));
}
```
The generated file shouldn't be the one to decide what module it is in.

Reply:
The reasoning is sound, but something about this is bugging me, as either it's unhygienic or there is simply no point in leaving the name choice to the downstream code. @JarvisCraft, WDYT?

Reply:
I am in favor of `blocks` module. It should not affect hygiene if implemented correctly (i.e., the generator should produce the valid results no matter where the include is located), eg., by providing extra information to it.